tests/installation/grub_test.pm

```diff
@@ -21,6 +21,7 @@ use grub_utils qw(grub_test);
 
 sub run {
     grub_test();
+    reset_consoles if (get_var('AGAMA_PRODUCT') eq 'Tumbleweed');
```

Comment:
you could use here:
```
if is_opensuse() && get_var('VERSION', '') =~ /agama/
```
which means it is tumbleweed and then you can change VERSION and reset console.
but I would prefer to not depend on the old code, as you can see it is really tricky to hack.

Reply:
We can just change the `VERSION` after the installation, if we set it at beginning it will boot into `package "Distribution::Opensuse::Tumbleweed` like [**this case**](https://openqa.opensuse.org/tests/4546932#step/boot_agama/2).